tools/gulptasks/dashboards/lint.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) Highsoft AS
+ */
+
+const gulp = require('gulp');
+
+/* *
+ *
+ *  Constants
+ *
+ * */
+
+const SOURCE_GLOB = './**/*.ts';
+
+/* *
+ *
+ *  Tasks
+ *
+ * */
+
+/**
+ * Lint test of TypeScript code.
+ *
+ * @return {Promise<string>}
+ *         Promise to keep with console output
+ */
+function task() {
+
+    const processLib = require('../lib/process');
+    const logLib = require('../lib/log');
+
+    return new Promise((resolve, reject) => {
+
+        logLib.message('Linting [./Dashboards/**/*.ts]...');
+
+        processLib
+            .exec('cd ts/Dashboards && npx eslint --quiet "' + SOURCE_GLOB + '"')
```

Comment:
Use the `cwd` option of `processLib.exec` instead of the `cd` command.
https://nodejs.org/api/child_process.html#child_processexeccommand-options-callback

You can also integrate the constant value here. It is not used anywhere else.